tools/docker/compose

```diff
@@ -12,32 +11,26 @@ else
   base_files="$base_files -f docker-compose.paritynet.yaml"
 fi
 
-base="docker-compose $base_files"      # base config, used standalone for acceptance
-dev="$base -f docker-compose.dev.yaml" # config for cldev
+base="docker-compose $base_files"
+dev="$base -f docker-compose.dev.yaml"
 
 clean_docker() {
   $base down -v --remove-orphans
   $dev down -v --remove-orphans
 }
 
-function usage() {
+usage() {
   echo "compose -- A helper script for running common docker-compose commands\
 
 Commands:
     help                  Displays this help menu
     clean                 Remove any containers and volumes related to compose files
     logs                  Display the logs of any service(s) by name
-
-    cld                   Runs the chainlink node container in dev mode
-
-    acceptance            Run the services required to perform an acceptance test
```

Comment:
why did you decide to remove this functionality?

Reply:
I haven't its just renaming as it was just a docker compose up behind and not running any acceptance test

Reply:
I see, that was a bad naming then.

Reply:
Could you please update the README file according then?

Reply:
Yes I'll be pushing soon the updates on the README, PR is still in draft